Modules/1002 IN/ARCaDIUS_IN2/ARCaDIUS_Serial.cpp

```diff
@@ -180,13 +186,15 @@ void ASerial::analyse() {
       Shutter();
       break;
     case 'R':
-      op = READ;
```

Comment:
Removing this means its not consistent with the other cases, if I'm reading this diff correctly.